crates/astria-sequencer/src/crypto.rs

```diff
@@ -1,39 +0,0 @@
-pub(crate) use ed25519_consensus::{
-    Signature,
-    SigningKey,
-    VerificationKey,
-};
-
-#[cfg(test)]
-mod test {
```

Comment:
it would be nice to keep this somewhere as it shows how the default accounts were created, but i agree it isn't a test and should probably not live here. thoughts?

Reply:
I will bring these back as doc tests or maybe as an example.

I'll think on it over night 😃